about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Browse Caitlin's abstract art">
+    <meta name="keywords" content="zentangles, abstract, art, drawing">
+
+    <title>Caitlin's Art</title>
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="container">
+      <div class="grey-background">
+        <a name="top"></a>
```

Comment:
name is deprecated in HTML5. Use id instead. http://www.w3schools.com/tags/att_a_name.asp